site/src/api/client.js

```diff
@@ -0,0 +1,11 @@
+import Axios from 'axios';
+
+export const BASE_URL =
+  process.env.LEADERBOARD_BASE_URL ||
+  'https://corsproxy.io/?https://discuss.layer5.io';
```

Comment:
did we required to add this flag for cors? I remember we enabled the cors from yaml file of discuss forum it allowed us to fetch api responses without any error

Reply:
So long as this evaluation works when the site is published and works for running locally, it can (should) be left so that the ongoing development experience isn’t more difficult than necessary.

Reply:
Copy that